test/clj/harja/kyselyt/kanavat/kanavan_hairiotilanne_test.clj

```diff
@@ -31,7 +31,7 @@
         hae-maara #(count (q "select id from kan_hairio where poistettu is not true;"))
         maara-alussa (hae-maara)
         kayttaja-id (:id +kayttaja-jvh+)
-        urakka-id (hae-saimaan-kanavaurakan-id)
```

Comment:
Tästä eteenpäin on valtavasti muutoksia jossa vain siistittiin koodia miten haetaan urakan id urakan nimellä.... vaikutti suureen määrään tiedostoja vaikka sinänsä yksinkertainen find and replace -operaatio.